superset/utils/pandas_postprocessing.py

```diff
@@ -916,3 +916,29 @@ def outliers(series: Series) -> Set[float]:
         for metric in metrics
     }
     return aggregate(df, groupby=groupby, aggregates=aggregates)
+
+
+def resample(
+    df: DataFrame,
+    resample_rule: str,
+    resample_method: str,
+    time_column: str,
+    resample_fill_zero: bool = False,
+) -> DataFrame:
```

Comment:
As these are all resample options, could we just remove the `resample_` prefix, resulting in `rule`, `method` etc?

Reply:
Okay. sure.